multicluster/controllers/multicluster/member/serviceexport_controller.go

```diff
@@ -626,10 +665,106 @@ func (r *ServiceExportReconciler) updateOrCreateResourceExport(resName string,
 	return nil
 }
 
+// getSubsetsFromEndpointSlice will check all ready Endpoints from any matched EndpointSlice
```

Comment:
"EndpointSlices"

Add "." to the end of the sentence.

Reply:
check all ready Endpoints from any matched EndpointSlice -> "get all ready endpoints from all the EndpointSlices which will be merged to one Endpoints."?